sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Microsoft.Azure.WebJobs.Extensions.SignalRService.csproj

```diff
@@ -11,24 +11,24 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!--Pin the versions of packages that may be upgraded during the process of Azure SDK moving all dependencies to the 8.x line to avoid incompatibility, until a compatible dependency gragh is figured out and finalized.-->
     <PackageReference Include="MessagePack" />
-    <PackageReference Include="Microsoft.Extensions.Azure" />
-    <PackageReference Include="Microsoft.Azure.Functions.Extensions" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" VersionOverride="3.0.37"/>
     <PackageReference Include="Microsoft.Azure.SignalR.Management" />
     <PackageReference Include="Microsoft.Azure.SignalR" />
     <PackageReference Include="Microsoft.Azure.SignalR.Protocols" />
     <PackageReference Include="Microsoft.Azure.SignalR.Serverless.Protocols" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" />
+    <PackageReference Include="Microsoft.Extensions.Azure" VersionOverride="1.10.0"/>
```

Comment:
This override is not necessary.   It matches the [central version](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L236).  

If you're releasing in the short term, you may need to move to the previous version in order to ensure compatibility.   I'd double-check with the functions team to be sure.   If sticking with 1.10.0, please remove the overrride.

Reply:
I'd prefer to stick to the central version. May I know whether we get confirmation from functions team that 1.10.0 is compatible with functions host?

Reply:
Issue created : https://github.com/Azure/azure-functions-host/issues/10875

Reply:
I don't think what you've chosen is the best path forward.  The Functions team does not have any knowledge of `Microsoft.Extensions.Azure` directly.   What you'll want to do is open an email with Fabio and team and talk about the dependency chain rather than a generic issue for a library they don't own.

Reply:
According to @amamounelsayed, the new ETA of function host v4.1037 will be March 21.  Before that, if we depend on Microsoft.Extensions.Azure 1.7.6 or above, we will meet issue https://github.com/Azure/azure-functions-host/issues/10575 . Therefore, I override its version to 1.7.3, a compatible version verified in SignalR extensions 1.14.1. Please havea a look.

Reply:
It would be better to go with 1.7.5, which is the lastest version of `Miccrosoft.Extensions.Azure` that still has a dependency on `System.Memory.Data` v1.0.2.    The move to 6.x for `System.Memory.Data` is what caused the incompatibility with the in-process host.

Reply:
Although @liliankasem suggested 1.7.9, the version is not listed on Nuget.Org and can't be restored.
![image](https://github.com/user-attachments/assets/057badbf-e7e1-4c27-93d3-dcf1fb96c5ec)
It seems no harm to use 1.7.5 as suggested by you , so I now use 1.7.5.

Reply:
Any dependency using System.Memory.Data 6.0.0 should be safe to use with v4.37.* of the host which is currenrly rolling out. From our perspective, using 1.7.6+ should be okay to release today.

If you'd like for us to evaulate the entire dependency of `Miccrosoft.Extensions.Azure` **1.10.0**, please provide a full dependency delta (including transitive dependencies). With that information, we should be able to spot anything that crossed a major version and check if those are host managed assemblies.

Reply:
Given Lillian's statement, I'd suggest going with 1.8.0, which references `System.Memory.Data` v6 and the patched `System.Text.Json`.      

Versions 1.9.0 and 1.10.0 have a bump for `Microsoft.Bcl.AsyncInterfaces` and `Microsoft.Extensions.DependencyInjection.Abstractions` into the 8.x line, which have not yet been approved by the Functions team.   If @liliankasem says  they're safe, then taking the latest is the way to go.

Reply:
Confirming that 1.8.0 is safe.

In the in-proc host `Microsoft.Bcl.AsyncInterfaces` and `Microsoft.Extensions.DependencyInjection.Abstractions` are both on 6.0.0,  so 1.9.0+ would break things for in-proc customers (despite being okay for out-of-proc host).

```
<    > Microsoft.Bcl.AsyncInterfaces                                                      6.0.0
---
>    > Microsoft.Bcl.AsyncInterfaces                                                      8.0.0

<    > Microsoft.Extensions.DependencyInjection.Abstractions                              6.0.0
---
>    > Microsoft.Extensions.DependencyInjection.Abstractions                              8.0.2
```

Reply:
@jsquire  Sorry not related to this PR, but more of a general comment: if `1.10.0` is the central version, does this mean we can't take on any new versions of the other WebJobs extensions, or are they also overrriding the central version for now?